lldb/source/Utility/ProcessInfo.cpp

```diff
@@ -117,6 +118,10 @@ bool ProcessInfo::IsScriptedProcess() const {
   return m_scripted_metadata_sp && *m_scripted_metadata_sp;
 }
 
+bool ProcessInstanceInfo::NiceValueIsValid() const {
+  return m_nice_value >= PRIO_MIN && m_nice_value <= PRIO_MAX;
```

Comment:
`PRIO_MIN` and `PRIO_MAX` come from linux/unix header and won't be available on other systems. I am also wondering if the `PRIO_MIN` and `PRIO_MAX` will differ from one linux OS to another? Probably best to not rely on these defines from OS specific header files, so I would suggest either manually defining them in this file or not checking the values.

Reply:
I removed this and just made the check to be a value stored in the optional